docs/resources/role.md

```diff
@@ -60,18 +60,16 @@ resource "powerscale_role" "role_test" {
 
   # Optional fields both for creating and updating
   description = "role_test_description"
-  # To add members, please provide uid/gid or provide name and type
+  # To add members, the uid/gid is required. Please use user/user_group datasource to look up the uid/gid needed.
   members = [
     {
-      name = "admin",
-      type = "user"
+      id = "UID:10"
```

Comment:
the id/name is convenient to user, we could keep them. And we can check name and id are matched when assign values

Reply:
Hi @taohe1012 ,as discussed in scrum, we shall keep the id as required now. Thanks

Reply:
this is not wrong error solution, but id/name is indeed more convenient to user.... approved